CDP4IME/ShellViewModel.cs

```diff
@@ -91,19 +97,30 @@ public class ShellViewModel : ReactiveObject, IDisposable
         /// </summary>
         private IDisposable subscription;
 
+        /// <summary>
+        /// Gets the <see cref="IAppSettingsService"/> that is used to read app settings
+        /// </summary>
+        private readonly IAppSettingsService appSettingService;
+
         /// <summary>
         /// Initializes a new instance of the <see cref="ShellViewModel"/> class.
         /// </summary>
         /// <param name="dialogNavigationService">
         /// The <see cref="IDialogNavigationService"/> that is used to show modal dialogs to the user
         /// </param>
-        public ShellViewModel(IDialogNavigationService dialogNavigationService)
+        /// <param name="appSettingService">
+        /// The (MEF injected) instance of <see cref="IAppSettingsService"/>
```

Comment:
this is not MEF injected in this place, the `(MEF injected)` can be removed from the docs

Reply:
Code has been refactored and moved to boostrapper